src/markdown-pages/automate-workflows/get-started-terraform.mdx

```diff
@@ -37,273 +45,324 @@ To use this guide, you should have some basic knowledge of both New Relic and Te
 
-To connect Terraform with New Relic, you need to set New Relic as a provider in your Terraform configuration file, so that Terraform can create and manage New Relic resources in your account.
-
-To set New Relic as a provider, create a Terraform configuration file (`main.tf`). This code sets `newrelic` as the Terraform provider and provisions an empty [alert policy](/docs/alerts/new-relic-alerts/configuring-alert-policies/create-edit-or-find-alert-policy) as the resource.
-
-```hcl lineNumbers=false
-provider "newrelic" {}
+<br />
 
-resource "newrelic_alert_policy" "alert_policy_name" {
-  name = "My Alert Policy Name"
+```hcl
+terraform {
+  # Require Terraform version 0.13.x (recommended)
+  required_version = "~> 0.13.0"
+
+  # Require version 2.11.x of the New Relic provider
+  required_providers {
+    newrelic = {
+      source  = "newrelic/newrelic"
+      version = "~> 2.11.0"
+    }
+  }
 }
 ```
 
+<Tip>
+<small>We recommend pinning your New Relic provider version by setting a <a target="_blank" href="https://www.terraform.io/docs/configuration/version-constraints.html">version constraint</a> similar to the example ("~> 2.11.0"). Using the right version constraint for your setup can provide better stability with your Terraform runs. </small>
+</Tip>
 </Step>
-<Step>
 
-## Initialize and test your setup
+<!-- PROVIDER CONFIGURATION -->
 
-After adding New Relic as a provider, [initialize](https://www.terraform.io/docs/commands/init.html) the working directory from the command line using `terraform init`.
-
-Once you've successfully initialized the working directory, test Terraform's [execution plan](https://www.terraform.io/docs/commands/plan.html) using `terraform plan` to confirm that you've the right setup.
+<Step>
 
-```bash lineNumbers=false
-# Initialize the working directory
-$ terraform init
+## Configure the New Relic provider
 
-# Test the Terraform plan
-$ terraform plan
-```
+With the `terraform` configuration block all set, we can configure the New Relic provider. Configuring the provider using the `provider` block is a quick way to us get started.
 
-`plan` performs a [dry run](<https://en.wikipedia.org/wiki/Dry_run_(testing)>) of your Terraform configuration, so nothing is actually provisioned. Always run `plan` to test your configuration before applying it.
+You will need the following items to configure the provider.
+1. Your [New Relic Account ID](https://docs.newrelic.com/docs/accounts/install-new-relic/account-setup/account-id).
+2. Your [New Relic Personal API key](https://docs.newrelic.com/docs/apis/get-started/intro-apis/types-new-relic-api-keys#personal-api-key). Most Personal API keys begin with the prefix `NRAK-`.
```

Comment:
I certainly wish it was "all", but we were advised not to use "all" in our other docs for this reason. AFAIK, legacy keys are the Admin API Key and generic API key. Personal API keys are the new standard for sure, but we were told there could be some Personal API keys out there that do not follow this standard. I'm all about clarity, and happy to rephrase, but maybe we confirm this if possible?

Reply:
Yeah, I think that makes sense, even in our product documentation, to explain that better. I can reach out to the docs team for an answer on this.

Reply:
So, the answer is: we're conforming to some INFOSEC guidelines and that *all* newly-generated keys will have prefixes. So, I think it would be more helpful to say something like:

> All personal API keys have an `NRAK-` prefix, unless it is a legacy key generated before August 1, 2020.

Reply:
Cool thanks.